build/stf-run-ci/tasks/clone_repos.yml

```diff
@@ -69,9 +69,11 @@
         repo: https://github.com/viaq/loki-operator
         dest: working/loki-operator
         version: "{{ loki_operator_branch | default(branch, true) }}"
+        force: yes
```

Comment:
I'm curious why these needed to be added.

Reply:
The stf-run-ci does a few changes to the cloned loki-operator while trying to deploy. If you don't use "force: yes", have any changes in the code and try to rerun the stf-run-ci, then ansible refuses to re-clone the repository and fails. The only solution is either to delete the loki-operator folder or use "force: yes". I don't expect we will do any development of the loki-operator here, so deleting the changes and recloning the operator shouldn't matter.

Reply:
Okay, so if I understand correctly, this is more of an issue in local testing, not a CI environment?

Reply:
It could affect the CI system as well depending on how it works (I'm not sure if a full repo removal and re-clone happens in the CI environment... I think an agent pod is probably spun up and re-clones everything fresh each time; @pleimer can you confirm that?)

If the CI environment is cleaned every time, then yea, this is more of a local issue. We could (should?) just wipe the `working` directory at the start of each of these runs. There is probably only a small amount of time saved by keeping the local working dirs.